format/mbr/mbr.go

```diff
@@ -0,0 +1,22 @@
+package mbr
+
+import (
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/registry"
+	"github.com/wader/fq/pkg/decode"
+)
+
+func init() {
+	registry.MustRegister(decode.Format{
+		Name:        format.MBR,
```

Comment:
Are there other computer platforms that use the name "mbr"? thinking maybe pc_mbr etc? but maybe mbr is synonomus with IBM PC MBR?

Reply:
Good question, I'll look if there are any collisions and if so, disambiguate

Reply:
I haven't found any other formats that use the "MBR" as their initialism. In fact, I learned that non-x86 machines like RPi use the same exact format but just ignore the code section and only use the partition table.

Reply:
Aha didn't know, so ignore code section by leaving it zero etc?

Yeap let's call it mbr 👍